src/LoadMask.js

```diff
@@ -87,30 +102,33 @@ export default function LoadMask({
 
 LoadMask.propTypes = {
   /**
-   * Set to true to display the load mask, otherwise it will be hidden.
-   * Defaults to false.
+   * Override or extend the styles applied to the component. See [CSS API](#css) below for more details.
+   */
+  classes: PropTypes.object,
+
+  /**
+   * Set to `true` to display the load mask, otherwise it will be hidden.
    */
   show: PropTypes.bool,
 
   /**
-   * Set to true to toggle the overflow style on the body when showing.
-   * Defaults to false
+   * Set to `true` to toggle the overflow style on the body when showing.
    */
   fullscreen: PropTypes.bool,
 
   /**
-   * Set to true to show partially background through the load mask
+   * Set to `true` to show partially background through the load mask
    */
   transparent: PropTypes.bool,
 
   /**
-   * Set to top to show the spinner near the top. Defaults to 'center'
+   * Set to `'top'` to show the spinner near the top.
    */
   align: PropTypes.oneOf(['center', 'top']),
 }
 
 LoadMask.defaultProps = {
-  show: null,
+  show: false,
```

Comment:
Should be null.

Reply:
I see why this matters now. I have removed the default since it's not necessary, and I've changed the prop description to reflect that specifically defining it changes its behavior. If I'm misunderstanding this, let me know.